.circleci/config.yml

```diff
@@ -87,6 +87,13 @@ jobs:
     steps:
       - checkout
       - run: make generate-check
+  "sdk release":
+    executor: golang
+    working_directory: /go/src/github.com/hashicorp/packer-plugin-sdk
```

Comment:
Do we need to set this working_directory on every job? I notice not all of them set it.

Reply:
I would guess that the job would be in the repo folder by default base on the ones that don't set this field.

Reply:
That's a good question. I don't suspect it is needed since we are using Go modules on for this project. I will adjust for this PR. Then open a new one to remove it from the other jobs once this is ready to go.